ReactWindows/ReactNative.Shared.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -103,7 +113,11 @@ public void NetworkingModule_Request_Content_String()
             var waitHandle = new AutoResetEvent(false);
             var module = CreateNetworkingModule(new MockHttpClient(request =>
                 {
+#if WINDOWS_UWP
                     var body = request.Content.ReadAsStringAsync().GetResults();
+#else
```

Comment:
No leave it for now, can you file an issue to investigate the UWP tests that use GetResults() though?

---
In reply to: [91598112](https://github.com/ReactWindows/react-native-windows/pull/899#discussion_r91598112) [](ancestors = 91598112)